packages/app-project/components/ClassifyPage/components/FinishedForTheDay/FinishedForTheDay.js

```diff
@@ -0,0 +1,56 @@
+import counterpart from 'counterpart'
+import { Box, Button, Grid, Heading, Paragraph } from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import en from './locales/en'
+import ProjectImage from './components/ProjectImage'
+import ClassifyBox from '../ClassifyBox'
+
+counterpart.registerTranslations('en', en)
+
+const StyledButton = styled(Button)`
+  border-width: 1px;
+  font-size: 14px;
```

Comment:
I've been passing the Grommet `Text` component into the `Button`'s label prop which has its own prop to set the text size. So something like:
```
<Button
  label={<Text size='small'>{counterpart('text.label')}</Text>}
/>
```

Reply:
nice 👍